pkgs/development/python-modules/jinja2-humanize-extension/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  humanize,
+  jinja2,
+}:
+
+buildPythonPackage rec {
+  pname = "jinja2-humanize-extension";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "metwork-framework";
+    repo = "jinja2_humanize_extension";
+    tag = "v${version}";
+    hash = "sha256-bSSwPCPLIWIRhIgaSwHnVTj5mpvwn259GXYeGr5NHBQ=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    humanize
+    jinja2
+  ];
+
+  pythonImportsCheck = [
```

Comment:
No tests ? Then add doChecks=false;

Reply:
I think this comment hasn't been addressed either

Reply:
I've added nativeCheckInputs pytestCheckHook on line 36.
the tests are passing. did I misunderstand your intent ?

Reply:
Sorry ignore my comment ! Validating this.